packages/transport-bridge/src/http.ts

```diff
@@ -182,12 +182,12 @@ export class TrezordNode {
                     ) {
                         next(req, res);
                     } else {
-                        allowOrigins(['https://sldev.cz', 'https://trezor.io', 'http://localhost'])(
-                            req,
-                            res,
-                            next,
-                            context,
-                        );
+                        allowOrigins([
+                            'https://sldev.cz',
+                            'https://trezor.io',
+                            'http://localhost',
+                            'http://suite.trezoriovpjcahpzkrewelclulmszwbqpzmzgub37gbcjlvluxtruqad.onion/web',
```

Comment:
Tor browser is not sending the real origin since `.onion` addresses are considered to be secret so it sends string "null" instead. https://gitlab.torproject.org/tpo/applications/tor-browser/-/issues/32865

It is still following standards like see  in: https://datatracker.ietf.org/doc/html/rfc6454#section-7.3

```
 Whenever a user agent issues an HTTP request from a "privacy-
   sensitive" context, the user agent MUST send the value "null" in the
   Origin header field.
```

Reply:
That's is why this flag is required in Tor `"network.http.referer.hideOnionSource - false"`

Reply:
Maybe we could create a Knowledge based documentation on how to use suite from Tor browser. @MiroslavProchazka

Reply:
We already have it - see https://trezor.io/learn/a/tor-in-trezor-suite

Reply:
Wow :open_mouth: ! That's great! And I did not realize it is in the description of this PR. :cry: